packages/synapse-react-client/src/components/Authentication/LastLoginInfo.tsx

```diff
@@ -0,0 +1,199 @@
+  const {
+    currentSourceAppName,
+    lastLoginMethod,
+    lastLoginSourceAppName,
+    lastLoginSourceAppURL,
+    lastLoginDate,
+    display,
+  } = props
+
+  const shouldNotShowLastLoginInfo =
+    currentSourceAppName === null ||
+    lastLoginMethod === null ||
+    lastLoginSourceAppName === null ||
+    lastLoginSourceAppURL === null ||
+    lastLoginDate === null ||
+    currentSourceAppName === lastLoginSourceAppName
+
+  if (shouldNotShowLastLoginInfo) {
+    return null
+  }
+
+  const currentSourceAppIndefiniteArticle =
+    getSourceAppIndefiniteArticle(currentSourceAppName)
+  const currentSourceAppDefiniteArticle =
+    getSourceAppDefiniteArticle(currentSourceAppName)
+  const lastLoginSourceAppDefiniteArticle = getSourceAppDefiniteArticle(
+    lastLoginSourceAppName,
+  )
+
+  const title = `You may already have ${currentSourceAppIndefiniteArticle} ${currentSourceAppName} account`
+
+  const tooltipText = (
+    <>
+      <Typography variant="smallText2" mb={1}>
+        {title}
+      </Typography>
+      <Typography variant="smallText1" mb={1} color="white">
+        {`This computer was used to sign in to an account on ${lastLoginSourceAppDefiniteArticle}${lastLoginSourceAppName}. 
+          You can use the same account to sign in to ${currentSourceAppDefiniteArticle}${currentSourceAppName}.`}
+      </Typography>
+      <Typography variant="smallText1" color="white">
+        {`Account last used on ${lastLoginDate}`}
+      </Typography>
+    </>
+  )
+
+  const boxText = (
+    <>
+      <Typography
+        variant="smallText2"
+        mb={1}
+        sx={{ color: 'grey.900', fontSize: '12px' }}
+      >
+        {title}
+      </Typography>
+      <Typography
+        variant="smallText1"
+        mb={1}
+        sx={{ color: 'grey.900', fontSize: '12px' }}
+      >
+        It looks like you already have an account, last used on{' '}
+        {lastLoginSourceAppDefiniteArticle}
+        <Link href={lastLoginSourceAppURL}>{lastLoginSourceAppName}</Link>.{' '}
+        {`You can use this account to sign in via ${getLoginMethodFriendlyName(
+          lastLoginMethod,
+        )}.`}
+      </Typography>
+      <Typography
+        variant="smallText1"
+        sx={{ color: 'grey.700', fontSize: '10px' }}
+      >
+        {`Account last used on ${lastLoginDate}`}
+      </Typography>
+    </>
+  )
+
+  return (
+    <>
+      {display === 'sentence' ? (
+        <Typography
+          variant="smallText1"
+          sx={{ color: 'grey.700', textTransform: 'none' }}
+        >
+          You may be able to use an existing account
+          <Tooltip
+            title={tooltipText}
+            placement={'top'}
+            componentsProps={{
+              tooltip: {
+                sx: {
+                  bgcolor: '#63676C',
+                  '& .MuiTooltip-arrow': {
+                    color: '#63676C',
+                  },
+                },
```

Comment:
Update the tooltip background color to match the design. Could also consider updating the default SRC theme, since the [SDS](https://www.figma.com/file/0oPm5lLSUva8kyfVNMS6FA/Sage-Style-%26-Component-Library?node-id=187%3A6615) appears to use this lighter tooltip background color as well

Reply:
Definitely! I created the ticket in the [SWC project](https://sagebionetworks.jira.com/browse/SWC-6507), since the change could impact SWC, but happy to move the ticket to PORTALS, if that's a better spot for it